setup.cfg

```diff
@@ -0,0 +1,24 @@
+[metadata]
```

Comment:
setup.cfg/py I think are only used by the doc build process, so they're not necessary